src/vsr/replica.zig

```diff
@@ -5763,6 +5760,12 @@ pub fn ReplicaType(
             return vsr.Checkpoint.trigger_for_checkpoint(self.op_checkpoint_next()).?;
         }
 
+        /// Returns the op that triggered the current checkpoint.
+        fn op_checkpoint_trigger(self: *const Self) u64 {
```

Comment:
I'd rather we not have this _particualr_ helper: we already have `op_checkpoint_next_trigger`, and it feels like confusing the two would be _very_ easy (mentally, I call "checkpoint trigger" the trigger that is in the future)

Reply:
Ah, good point... I haven't been able to find a crisp terminology for this yet as well! When I'm thinking to myself, I usually call `self.op_checkpoint()` as the *current checkpoint* and  `self.op_checkpoint()` as the *next checkpoint*, but I do find myself getting confused a fair few times if I'm not super attentive. 😅

Reply:
https://github.com/tigerbeetle/tigerbeetle/pull/2236/commits/cecee4531bb3588085d35ecf1caf80d17da0995a